Makefile

```diff
@@ -3,13 +3,17 @@ all: dependencies test
 base_filename=kata
 
 dependencies:
-	which mit-scheme >/dev/null 2>&1 || brew install mit-scheme
```

Comment:
Why remove `>/dev/null 2>&1`? I used it because I thought the output of `which` was useless and distracting, since `which` is only being used for its return code, not for its output. I also wanted `make --silent` to output nothing but the test results.